consensus/polybft/sc_integration_test.go

```diff
@@ -323,47 +326,26 @@ func TestIntegration_CommitEpoch(t *testing.T) {
 	}
 }
 
-func deployRootchainContract(t *testing.T, transition *state.Transition, rootchainArtifact *artifact.Artifact, sender types.Address, accSet AccountSet, bn256Addr types.Address) types.Address {
+func deployAndInitContract(t *testing.T, transition *state.Transition, scArtifact *artifact.Artifact, sender types.Address,
+	initCallback func() ([]byte, error)) types.Address {
 	t.Helper()
 
-	result := transition.Create2(sender, rootchainArtifact.Bytecode, big.NewInt(0), 1000000000)
+	result := transition.Create2(sender, scArtifact.Bytecode, big.NewInt(0), 1e9)
 	assert.NoError(t, result.Err)
-	rcAddress := result.Address
-
-	initialize := contractsapi.InitializeCheckpointManagerFunction{
-		NewBls:          contracts.BLSContract,
-		NewBn256G2:      bn256Addr,
-		NewValidatorSet: accSet.ToAPIBinding(),
-		ChainID_:        big.NewInt(0),
-	}
-
-	init, err := initialize.EncodeAbi()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	result = transition.Call2(sender, rcAddress, init, big.NewInt(0), 1000000000)
-	require.True(t, result.Succeeded())
-	require.False(t, result.Failed())
-	require.NoError(t, result.Err)
+	addr := result.Address
 
-	return rcAddress
-}
+	var (
```

Comment:
Nitpick: You don't need this var block, since you are using them in one if block, so just initialize them there.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1259/commits/05613644c9cbc998370fba4a89ea67499e7a503a